sites/cheerpx/src/content/docs/12-reference/21-index.list.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Index.list
+description: Purpose and usage of index.list
+---
+
+## What is `index.list`
```

Comment:
Having a separate reference seems to me excessively verbose and a poor fit in the current API reference structure. 
![image](https://github.com/user-attachments/assets/86acc952-74d7-470c-bc1b-f34441a83d98)

`index.list` is not an API, why should it be there?

Let's try something simpler. The example and screenshots are certainly not required. Add a new section to the WebDevice reference instead with a shorted version of this guide.

The most important thing that is' missing is to explain people _how_ to create this file. Try to build a `find` oneliner that creates the file. Keep in mind that you cannot simply do `ls > index.list` since `index.list` will then spuriously appear in the listing. The combination of `ls > .index.list` + `mv .index.list index.list` might work.